app/helpers/api/dapi/getDocumentByFileName.test.js

```diff
@@ -29,7 +29,7 @@ describe('Document controller', () => {
 
       expect(getDocument.mock.calls.length).toEqual(1);
       expect(getDocument).toHaveBeenCalledWith({
-        endpoint: 'http://localhost:5101/api/v1/documents/a-document.pdf',
+        documentEndpoint: 'http://localhost:5101/api/v1/documents/a-document.pdf',
         logger,
       });
     });
```

Comment:
In this file remove the first describe level `describe('Document controller', () => {` don't need that anymore